stripe/src/main/java/com/stripe/android/cards/CardAccountRangeSource.kt

```diff
@@ -4,8 +4,4 @@ import com.stripe.android.model.CardMetadata
 
 internal interface CardAccountRangeSource {
     suspend fun getAccountRange(cardNumber: String): CardMetadata.AccountRange?
```

Comment:
Does this function need to suspend?

Reply:
Yeah, because source can be a network, disk, or static resource.